rfc/system/0001-flyte-execution-tags.md

```diff
@@ -58,19 +59,11 @@ flytectl get execution -p flytesnacks -d development --filter.labels="k:v"
 ```
 
 ### UI Interface
+All labels are treated the same way and allow search/filter and click based grouping.
+Users will get the regular executions view with all the labels available on each execution.
+The users are allowed to filter an exection simply by clicking on a label and then all
+executions are filtered by that label. 
 
-Two approaches
```

Comment:
In the previous contributors synch we discussed that option 1 is preferable since we don't want to impose on users which label keys are considered special in their domain.